common/button/button-en.html

```diff
@@ -0,0 +1,415 @@
+{
+	"title": "Buttons",
+	"language": "en",
+	"description": "Buttons page including working examples to test how various button styles appear and function",
+	"dateModified": "2023-06-09"
```

Comment:
Missing page language alternative

Reply:
Addressed in commit  "Button: Addressed Review Comments"